crank/bin/mcd_calculate.py

```diff
@@ -0,0 +1,166 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# mcd_calculate.py
+# Copyright (C) 2020 Wen-Chin HUANG
+#
+# Distributed under terms of the MIT license.
+#
+
+import json
+import os
+import sys
+import argparse
+import logging
+import multiprocessing as mp
+import numpy as np
+import pysptk
+import pyworld as pw
+import scipy
+from fastdtw import fastdtw
+
+from pathlib import Path
+import soundfile as sf
+from parallel_wavegan.utils import find_files
+from sprocket.speech import FeatureExtractor
+from crank.net.trainer.dataset import read_feature
+from crank.utils import load_yaml, open_featsscp
+from crank.utils import low_cut_filter
+from crank.feature import Feature
+
+def get_world_features(cvt_path, trgspk, conf, spkr_conf):
+    x, fs = sf.read(str(cvt_path))
+    x = np.array(x, dtype=np.float)
+    x = low_cut_filter(x, fs, cutoff=70)
+    fe = FeatureExtractor(
+        analyzer="world",
+        fs=conf["feature"]["fs"],
+        fftl=conf["feature"]["fftl"],
+        shiftms=conf["feature"]["shiftms"],
+        minf0=spkr_conf[trgspk]["minf0"],
+        maxf0=spkr_conf[trgspk]["maxf0"],
+    )
+    cvt_f0, _, _ = fe.analyze(x)
+    cvt_mcep = fe.mcep(dim=conf["feature"]["mcep_dim"],
+                       alpha=conf["feature"]["mcep_alpha"])
+    return cvt_mcep, cvt_f0
+
+def get_basename(path):
+    return os.path.splitext(os.path.split(path)[-1])[0]
```

Comment:
Could you replace by ```Path``` because I prefer to use it in crank.

Reply:
Okay, let me learn how to use `Path` ...